components/CatalogLayout.vue

```diff
@@ -72,7 +72,11 @@
                 }
             "
         />
-        <CatalogContainer class="catalog__container" :assets="selectedCategoryAssets"></CatalogContainer>
+        <CatalogContainer
+            class="catalog__container"
+            :assets="selectedCategoryAssets"
+            @element-added="$emit('element-added')"
```

Comment:
The element-added event will automatically go through to outer components. There is no need to re-emit it.

Reply:
It does not in the end, it breaks the emit pattern if I remove this line.